modelzoo/DBMTL/train.py

```diff
@@ -0,0 +1,709 @@
+import time
+import argparse
+import numbers
+import tensorflow as tf
+import os
+import sys
+import math
+import collections
+from tensorflow.python.client import timeline
+import json
+from tensorflow.python.ops import partitioned_variables
+
+if tf.__version__ >= '2.0':
```

Comment:
Have you checked whether the script works on tf2? It might not due to `tf.data.Iterator`.

Reply:
Removed